Tests/Functional/Subscribers/LifecycleTest.php

```diff
@@ -7,44 +7,44 @@
 
 namespace ShopwarePlugins\Connect\Tests\Functional\Subscribers;
 
-use ShopwarePlugins\Connect\Tests\DatabaseTestCaseTrait;
-use Shopware\Components\Model\ModelManager;
-
-class LifecycleTest extends \Enlight_Components_Test_Plugin_TestCase
+use Doctrine\DBAL\Connection;
+use Shopware\Models\Article\Article;
+use Shopware\Models\Article\Detail;
+use ShopwarePlugins\Connect\Tests\TestClient;
+use ShopwarePlugins\Connect\Tests\WebTestCaseTrait;
+use Symfony\Component\HttpFoundation\Response;
+
+class LifecycleTest extends \PHPUnit_Framework_TestCase
 {
-    use DatabaseTestCaseTrait;
+    use WebTestCaseTrait;
```

Comment:
does the webTestCaseTrait roll back changes to the db ->
because if you invoke the backend controllers something could be changed in the db
even if you do no db-queries in the testCase